mm2src/mm2_main/Cargo.toml

```diff
@@ -12,15 +12,18 @@ edition = "2018"
 doctest = false
 
 [features]
-custom-swap-locktime = [] # only for testing purposes, should never be activated on release builds.
+custom-swap-locktime = [
+] # only for testing purposes, should never be activated on release builds.
```

Comment:
I think this is very bad way of formatting things.

Reply:
my editor is the culprit here.. thanks fixed.